rag_experiment_accelerator/config/path_config.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass
+import os
+from rag_experiment_accelerator.config import paths
+from rag_experiment_accelerator.config.base_config import BaseConfig
+
+
+@dataclass
+class PathConfig(BaseConfig):
+    artifacts_dir: str = ""
+    data_dir: str = ""
+    eval_data_file: str = ""
+    eval_data_dir: str = ""
+    generated_index_names_file: str = ""
+    query_data_dir: str = ""
+    sampling_output_dir: str = ""
+
+    def initialize_paths(self, config_file_path: str, data_dir: str) -> None:
+        self._config_dir = os.path.dirname(config_file_path)
+
+        if not self.artifacts_dir:
+            self.artifacts_dir = os.path.join(self._config_dir, "artifacts")
```

Comment:
Should these values (I mean "artifacts") and so on be moved to constants - is a questionable topic, but it will be easier to read imo